api/python/provisioner/commands/bootstrap_provisioner.py

```diff
@@ -634,16 +634,16 @@ def _run(self, nodes, **kwargs):  # noqa: MC0001, C901 FIXME
             paths['setup_key_file'],
             paths['salt_roster_file']
         )
-        if run_args.bootstrap_key:
-            self._prepare_roster(
-                run_args.nodes,
-                run_args.bootstrap_key,
-                paths['salt_bootstrap_roster_file']
-            )
-
-        if not run_args.field_setup:
-            logger.info("Copying config.ini to file root")
-            self._copy_config_ini(run_args, paths)
+        # if run_args.bootstrap_key:
```

Comment:
Instead of commenting, can we put it in some condition (example: based on bootstrap_key)

Reply:
Actually, it's not required anymore, it's not supported with the new flow. I was planning to remove it but kept it to see if it impacts in any way, will remove it completely once we have a few clean deployments.